osgi-test/build.gradle

```diff
@@ -66,6 +66,8 @@ test {
   systemProperty 'ehcache.osgi.jar', project(':dist').jar.archivePath.getPath()
   systemProperty 'ehcache.osgi.jcache.version', parent.jcacheVersion
   systemProperty 'ehcache.osgi.slf4j.version', parent.slf4jVersion
+}.doFirst {
```

Comment:
Looked at this briefly again, fixing it is a little more complicated.  I currently believe having one version of these tests thats work on both 1.6 and 9 is going to be impossible.  I'll let @ljacomet  decide what he wants to do here.